homeassistant/components/jvc_projector/entity.py

```diff
@@ -0,0 +1,38 @@
+"""Base Entity for the jvc_projector integration."""
+
+from __future__ import annotations
+
+import logging
+
+from jvcprojector import JvcProjector
+
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN, MANUFACTURER, NAME
+from .coordinator import JvcProjectorDataUpdateCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class JvcProjectorEntity(CoordinatorEntity[JvcProjectorDataUpdateCoordinator]):
+    """Defines a base JVC Projector entity."""
+
+    def __init__(self, coordinator: JvcProjectorDataUpdateCoordinator) -> None:
+        """Initialize the entity."""
+        super().__init__(coordinator)
+
+        self._attr_unique_id = coordinator.unique_id
+        self._attr_name = NAME
```

Comment:
If we want to to be the same as the device name, just don't set it on the entity level.

```suggestion
```